revup/topic_stack.py

```diff
@@ -54,6 +54,8 @@ def format_remote_branch(uploader: str, base_branch: str, topic: str) -> str:
 
 RE_COMMIT_LABEL = re.compile(r"^(?P<label1>[a-zA-Z\-_0-9]+):.*|^\[(?P<label2>[a-zA-Z\-_0-9]+)\].*")
 
+RE_BRANCH_ALLOWED = re.compile(r"^[a-zA-Z\d_\.\/-]+$")
```

Comment:
`\w` instead of `a-zA-Z\d` so unicode can be used

Also, comment on where this restriction comes from (valid branch names?)

Reply:
hmm i haven't tested with unicode but let me try that

Reply:
(commented)

Reply:
hmm github doesn't seem super happy but it did work https://github.com/Skydio/revup/pull/106 so i'll go with this